common/logisticspipes/utils/string/StringUtils.java

```diff
@@ -110,10 +110,12 @@ public static String getFormatedStackSize(long stackSize, boolean forceDisplayNu
 			s = stackSize + "";
 		} else if (stackSize < 100000) {
 			s = stackSize / 1000 + "K";
-		} else if (stackSize < 1000000) {
-			s = "0M" + stackSize / 100000;
-		} else {
+		} else if (stackSize < 10000000) {
+			s = (int)stackSize / 1000000 + "M" + (stackSize - ((int)stackSize / 1000000) * 1000000) / 100000;
```

Comment:
Instead of printing for example "10M2", I'd rather have it print "10.2M", that's more intuitive to read.

Maybe even always round to min. 3 significant digits to keep it consistent? So it'd show "1.23K", "46.1K", "71.1M", "5.62G", etc.

Reply:
It cant be more than 3 characters otherwise it will clip in the screen.
And I just left it as is because it is how it was done before, change what you want :)

Reply:
The problem with that solution is that we cannot have any decimal points because they count as a character meaning it would always clip what could also be changed even though is kind of unrelated is the font scale, I thought about it but it wasn't in the scope of the problem

Reply:
Oh right, this text gets put under the item stack. Yeah that won't fit more than 4 chars unfortunately. Was just a thought to make it read better, but if it's not easily possible then nevermind.